### PR TITLE
Premine error

### PR DESCRIPTION
**Pull request by leogaggl**

### CODE REVIEW

File: 0092-premine-error-correction.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ The payment awarded on 20th April, 2023 was 5,789.755 IOT tokens based on the ca
 This HIP proposes that the following process is followed after this HIP is approved by the community.
 
 1. A multi-sig premine correction wallet will be created to fund awarding qualifying Hotspots that were not awarded during the IOT pre-mine.
-2. Each qualifying Hotspot will be awarded 5,789.755 IOT tokens out of the multi-sig wallet and transferred to the wallet that owned the Hotspot at the time of the original pre-mine award (20th April, 2023). The specifics of how this is implemented will be 
+2. Each qualifying Hotspot will be awarded 5798.755355 (2_500_000_000/431_127) IOT tokens out of the multi-sig wallet and transferred to the wallet that owned the Hotspot at the time of the original pre-mine award (20th April, 2023). The specifics of how this is implemented will be
```
Comment (abhay) [inline 0092-premine-error-correction.md:31]:
```suggestion
2. Each qualifying Hotspot will be awarded 5,798.755355 (2,500,000,000 available IOT tokens / 431,127 rewarded Hotspots at migration) IOT tokens out of the multi-sig wallet and transferred to the wallet that owned the Hotspot at the time of the original pre-mine award (20th April, 2023). The specifics of how this is implemented will be 
```

Comment (abhay) [reply 0092-premine-error-correction.md:31]: slightly more human readable.

Comment (leogaggl) [reply 0092-premine-error-correction.md:31]: > slightly more human readable.

Agreed! 👍

Comment (abhay) [reply 0092-premine-error-correction.md:31]: oddly, doesn't look like this change actually committed?

**Review by abhay — APPROVED** [review]

**Pull request merged**